lib/aurora_bootstrap_parallelization/exporter.rb

```diff
@@ -3,10 +3,11 @@
 
 module AuroraBootstrapParallelization
   class Exporter
-    def initialize( config )
+    def initialize( config, is_cron_job )
```

Comment:
can we make this be 2 separate classes, one for `Job` and one for `CronJob`?

Reply:
Is this variable used? Do we still need it now that it's split into dedicated classes?